SETUP/DEVELOPMENT.md

```diff
@@ -0,0 +1,106 @@
+# Code Development
+
+This doc is designed for users who want to develop on the DProofreaders (DP)
+codebase.
+
+## Introduction
+
+The DP code is almost entirely server-side PHP with a smattering of JS and
+CSS. It uses a MySQL backend database and requires a web server -- pgdp.net
+uses Apache but Nginx should work as well. See the [installation docs](INSTALL.md)
+for information on minimum middleware versions.
+
+The code does not use a PHP framework or an ORM. It was developed 20 years ago
+and has grown very organically over the years. There has been effort in the
+past decade to standardize things but you'll find many different patterns remain.
+We have a defined [coding style](CODE_STYLE.md) for the codebase and a set of
+[best practices](https://www.pgdp.net/wiki/DP_Code_Best_Practices) that cover
+some of our prescribed patterns.
+
+## Contributing
+
+DP code uses git for source control and is housed at GitHub in the
+[DProofreaders/dproofreaders](https://github.com/DistributedProofreaders/dproofreaders)
+repository -- likely where you got this code to begin with.
+
+We use the "fork, branch, develop, merge" model, where developers are expected to:
+1. fork the main repository
+2. create a development branch off master
+3. develop code on the branch
+4. generate a merge/pull request when ready to merge
```

Comment:
Is there a set of automated regression tests a developer should run on their own before submitting a PR? Anything in the SETUP/tests directory that should be run before a PR?